08-interop/README.md

```diff
@@ -281,3 +281,188 @@ guide on Zephyr's documentation.
    dst_l2addr: 32:A9:FA:65:10:6B:11:14
    ~~ PKT    -  4 snips, total size:  85 byte
    ```
+   
+Task #12 - GNRC Border Router with WiFi uplink to the Internet
+========================
+### Description
+
+This test will ensure RIOT can connect to a real IPv6 enabled WiFi network and
+share the uplink connection with constrained nodes.
+
+An esp* board with the `gnrc_border_router` connects to your WiFi.
+The module `sock_dns` is used to resolve domain names.
+
+Two network interfaces are configured:
+
+ - a 802.11 WiFi interface used as uplink
+ - a proprietary `esp_now` interface as 6LoWPAN downlink
+ 
+ Materials
+---------
+
+You'll need:
+
+ - a pair of esp8266/esp32 boards
+ - 2.4 GHz WiFi network with an IPv6 uplink
+ - to make sure prefix delegation (IA_PD) is enabled in your router's DHCPv6 server
+   (On the popular Fritz!Box line of routers this can be enabled in the
+   Network -> Network Settings -> IPv6 Addresses menu)
+
+### Details
+
+Flash the `gnrc_border_router` example onto one of the esp* boards.
+Use the `sock_dns` module to enable name resolution.
+The credentials for the WiFi network will be passed on the command line.
+Replace `esp8266-esp-12x` with the esp* board of your choice, adjust `PORT` if needed.
+
+```
+USEMODULE=sock_dns make -C examples/gnrc_border_router BOARD=esp8266-esp-12x UPLINK=wifi WIFI_SSID='your_ssd' WIFI_PASS='your_password' PORT=/dev/ttyUSB0 flash term
+```
+
+### Result
+
+RIOT should be able to connect to your WiFi network and configure a global address on both interfaces
+
+```
+2020-08-02 20:38:20,346 # Iface  11  HWaddr: EC:FA:BC:5F:82:91  Channel: 6  Link: up 
+2020-08-02 20:38:20,349 #           L2-PDU:1500  MTU:1492  HL:255  RTR  
+2020-08-02 20:38:20,352 #           Source address length: 6
+2020-08-02 20:38:20,355 #           Link type: wireless
+2020-08-02 20:38:20,360 #           inet6 addr: fe80::eefa:bcff:fe5f:8291  scope: link  VAL
+2020-08-02 20:38:20,368 #           inet6 addr: 2001:16b8:453f:1f00:eefa:bcff:fe5f:8291  scope: global  VAL
+2020-08-02 20:38:20,371 #           inet6 group: ff02::2
+2020-08-02 20:38:20,374 #           inet6 group: ff02::1
+2020-08-02 20:38:20,377 #           inet6 group: ff02::1:ff5f:8291
+2020-08-02 20:38:20,377 #           
+2020-08-02 20:38:20,382 # Iface  10  HWaddr: EE:FA:BC:5F:82:91  Channel: 6 
+2020-08-02 20:38:20,385 #           L2-PDU:249  MTU:1280  HL:64  RTR  
+2020-08-02 20:38:20,390 #           RTR_ADV  6LO  Source address length: 6
+2020-08-02 20:38:20,393 #           Link type: wireless
+2020-08-02 20:38:20,399 #           inet6 addr: fe80::ecfa:bcff:fe5f:8291  scope: link  VAL
+2020-08-02 20:38:20,404 #           inet6 addr: 2001:16b8:453f:1ff0:ecfa:bcff:fe5f:8291  scope: global  VAL
+2020-08-02 20:38:20,407 #           inet6 group: ff02::2
+2020-08-02 20:38:20,410 #           inet6 group: ff02::1
+2020-08-02 20:38:20,416 #           inet6 group: ff02::1:ff5f:8291
+```
+
+You should be able to ping global addresses
```

Comment:
*if your provider supports IPv6 *EuroDOCSIS 3.0 modem* :eyes: Maybe this should be added as a disclaimer to the beginning of the testing description

Reply:
It lists

 - 2.4 GHz WiFi network with an IPv6 uplink

as a requirement.
I can make it more explicit, but hopefully the problem will [resolve itself](https://www.google.com/intl/en/ipv6/statistics.html) eventually.

Reply:
> I can make it more explicit, […]

Yes please. Testers can make themselves very stupid if they want to ;-).